target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -482,7 +482,8 @@ ath79_setup_macs()
 		label_mac=$lan_mac
 		;;
 	dlink,dap-2230-a1|\
-	dlink,dap-2660-a1)
+	dlink,dap-2660-a1|\
+	dlink,dap-2680-a1)
```

Comment:
You seem to have one-ports here, so please set them (all) up properly in 02_network.